Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -725,7 +725,9 @@ WarpX::PushPSATD ()
             PSATDSubtractCurrentPartialSumsAvg();
 
             // Synchronize J and rho (if used)
-            SyncCurrent(current_fp, current_cp);
+            // TODO This works only without mesh refinement
```

Comment:
Could you add a comment here saying that we use `SumBoundaryJ` instead of `SyncCurrent` because the current was already filtered in the `Evolve` function?